plugins/button/plugin.js

```diff
@@ -331,19 +328,22 @@
 			if ( element ) {
 				element.setState( state, 'cke_button' );
 
-				state == CKEDITOR.TRISTATE_DISABLED ?
-					element.setAttribute( 'aria-disabled', true ) :
+				if ( state == CKEDITOR.TRISTATE_DISABLED ) {
```

Comment:
In fact it could be even simplified to switching true/false value for the attribute. There is no need to remove it.

This is the case for all true/false ARIA attributes.